chapter2/tests/t6/chapter2-l2-t6-get-file-unique-dirs.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Not nice output when no argument passed
![image](https://user-images.githubusercontent.com/26448672/231134896-3fe6de45-2b1c-4867-9d94-3540620c0e25.png)

Reply:
1. Add *dir* input arg check.
2. Show errors of `file` command
3. Use global "dir".
4. Use `sort -u`